installation/tarball/provider_test.go

```diff
@@ -77,18 +78,28 @@ var _ = Describe("Provider", func() {
 
 			Context("when tarball is not present in cache", func() {
 				var (
-					tempDownloadFilePath string
+					tempDownloadFilePath1 string
+					tempDownloadFilePath2 string
+					tempDownloadFilePath3 string
 				)
 
 				BeforeEach(func() {
-					tempDownloadFile, err := ioutil.TempFile("", "temp-download-file")
```

Comment:
This test fails everywhere now without 3 temp files, as the file is being closed on each [retry][1]. The real file system is creating a new temp file on each of the 3 retries, but the test was previously only using one temp file for all retries.

[1]: https://github.com/cloudfoundry/bosh-cli/pull/171/files/f78baee58902f252afde22804b19549c0618b805#diff-860904024a4888f514f430f2bffc5c41R120